application/controllers/handler.js

```diff
@@ -49,10 +49,24 @@ module.exports = {
       const fileType = '.jpeg';
       const filePath = path.join(conf.fsPath, 'slideThumbnails/' + fileName + fileType);
       const html = request.payload;
+      let document = cheerio.load(html);
+      let pptxwidth = document('div[class=pptx2html]').css().width.replace('px', '');
+      let pptxheight = document('div[class=pptx2html]').css().height.replace('px', '');
+      pptxwidth = pptxwidth ? pptxwidth : 0;
```

Comment:
That's at least more simple to me, as you would save 4 lines instead, that I need to read otherwise --> I'm always lazy ;)